Makefile

```diff
@@ -1,28 +1,25 @@
-all: vendor
+all:
 	go build -o cni-noop ./cmd/cni-noop
 	go build -o cnispawn ./cmd/cnispawn
 	go build -o kubeadm-nspawn ./cmd/kubeadm-nspawn
 
-glide:
-	@which glide || go get -u github.com/Masterminds/glide
-
+vendor: glide.lock | glide
+	glide --quiet install --strip-vendor
 glide.lock: glide.yaml | glide
 	glide update --strip-vendor
+glide:
+	@which glide || go get -u github.com/Masterminds/glide
 
-vendor: glide.lock | glide
-	glide install --strip-vendor
-
-.PHONY: glide vendor all
-
-clean:
-	rm -rf ./{kubeadm-nspawn,cni-noop,cnispawn}
+.PHONY: vendor all clean
 
+clean: clean-bins clean-rootfs clean-cni clean-image clean-ssh-keys
+clean-ssh-keys:
```

Comment:
"clean-ssh-keys" is not a file, so it should be added in the `.PHONY` list.

See https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html#Phony-Targets:
> A phony target is one that is not really the name of a file